Translation/Translator.php

```diff
@@ -51,12 +52,20 @@ public function addResource($format, $resource, $locale, $domain = null)
     }
 
     /**
-     * Return the list of 'standard' resources
+     * Return the list of 'standard' resources.
      *
      * @return array
      */
     public function getStandardResources()
     {
+        if (empty($this->standardResources)) {
+            foreach ($this->resourceFiles as $file) {
+                // filename is domain.locale.format
+                list($domain, $locale, $format) = explode('.', basename($file), 3);
```

Comment:
travis is not happy with this: https://travis-ci.org/liip/LiipTranslationBundle/jobs/55473320

PHP Warning:  basename() expects parameter 1 to be string, array given in /home/travis/build/liip/LiipTranslationBundle/Translation/Translator.php on line 64